.gitignore

```diff
@@ -10,5 +10,6 @@
 /misc
 /notes.txt
 /pkg
-/tmp
+/tmp/*
+!/tmp/.keep
```

Comment:
> !/tmp/.keep

What this do?

Reply:
"Unignores" tmp/.keep, which was ignored by the previous line.